### PR TITLE
setup files css and components,

### PR DESCRIPTION
**Pull request by EddieJorden**

### CODE REVIEW

File: src/Components/Playlist/PlayList.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import TrackList from './TrackList';
+import './PlayList.class';
+
```
Comment (VarTheVar) [inline src/Components/Playlist/PlayList.js:3]:
```suggestion
import React from "react";
```

File: src/Components/SearchResults/SearchResults.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import TrackList from './TrackList.js';
+import './SearchResults.css';
+
```
Comment (VarTheVar) [inline src/Components/SearchResults/SearchResults.js:3]:
```suggestion
import React from "react";
```

File: src/Components/TrackList/TrackList.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import './TrackList.css';
+
```
Comment (VarTheVar) [inline src/Components/TrackList/TrackList.js:2]:
```suggestion
import React from "react";
```

File: src/Components/Track/Track.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import './Track.css'
```
Comment (VarTheVar) [inline src/Components/Track/Track.js:1]: This whole file is messy. Not sure what you want it to look like but clean up before PR

**Review by VarTheVar — CHANGES_REQUESTED** [review]
When you make a PR, all your code should be clean and ready for deploying. If it's not ready, don't make a PR. You can make commits, and push them up if you want to save them on github. But keep working on that branch until the code you're working on is ready before moving on. 


You missed some react imports and there's a file that needs cleaning up. Don't make a PR with linter errors. In fact never commit anything with linter errors for now

File: src/Components/Playlist/PlayList.js (hunk 1 above)
Comment (EddieJorden) [reply src/Components/Playlist/PlayList.js:3]: I would say this has been resolved --ej--

**Pull request closed without merging**